src/frontend/src/common/api/index.tsx

```diff
@@ -60,10 +78,24 @@ function convert<U extends APIResponse, T extends U[keyof U]>(
   const converted = jsonToType<T>(entry, keyMap);
   // key should always be a string anyways. no funky business please.
   converted.type = API_KEY_TO_TYPE[String(key)];
+  // ^^^ FIXME (Vince) Does this get used anywhere? What was original purpose?
```

Comment:
I wrote that, but I don't remember why!

Reply:
Okay, I'll make a note to see about removing this in an upcoming PR. Thanks!